src/enclave_proc/resource_manager.rs

```diff
@@ -481,19 +484,18 @@ impl EnclaveHandle {
                 )
             })?
             .len();
-        if eif_size > requested_mem {
+        if NE_EIF_MEM_THRESHOLD * eif_size > requested_mem {
             return Err(new_nitro_cli_failure!(
                 &format!(
                     "At least {} MiB must be allocated (which is the EIF file size)",
```

Comment:
The message does not seem to be up to date anymore.

Reply:
Right, updated in the latest revision.